RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/framework/SearchClusterContainer.java

```diff
@@ -59,6 +59,11 @@ public class SearchClusterContainer extends GenericContainer<SearchClusterContai
         "opensearchproject/opensearch:2.19.1",
         Version.fromString("OS 2.19.1")
     );
+    public static final ContainerVersion OS_V3_0_0 = new OpenSearchVersion(
+        "opensearchproject/opensearch:3.0.0",
+        Version.fromString("OS 3.0.0")
+    );
+    
     public static final ContainerVersion OS_LATEST = OS_V2_19_1;
```

Comment:
I'm curious if you think we should update this value at this point?

Reply:
Maybe after its a little more battle tested.  I think 2.19.* will be our version of choice for the next 2-3 months.